app/scripts/templates/settings/security.mustache

```diff
@@ -0,0 +1,87 @@
+<div id="security" class="settings-unit {{#isPanelOpen}}open{{/isPanelOpen}} security-panel">
+    <div class="settings-unit-stub">
+        <header class="settings-unit-summary">
+            <h2 class="settings-unit-title">{{#t}}Two-step authentication{{/t}}</h2>
+        </header>
+
+        {{^hasToken}}
+            <button class="settings-button primary settings-unit-toggle totp-create" data-href="settings/security">
+                {{#t}}Enable…{{/t}}
+            </button>
+        {{/hasToken}}
+
+        {{#hasToken}}
+            <button class="settings-button secondary settings-unit-toggle" data-href="settings/security">
+                {{#t}}Change…{{/t}}
+            </button>
+        {{/hasToken}}
+    </div>
+
+
+    <div class="settings-unit-details">
+        <form novalidate>
+            <p>
+                {{#t}}Add an extra layer of security to your account by requiring security codes.{{/t}}
+            </p>
+
+            <ul id="security-status" class="security-list">
+                <li class="security-type">
+                    <div class="name">Current status</div>
+                    <div class="details">
+                        {{#hasToken}}
+                            <div class="enabled">{{#t}}Enabled{{/t}}</div>
+                        {{/hasToken}}
+                        {{^hasToken}}
+                            <div class="disabled">{{#t}}Disabled{{/t}}</div>
+                        {{/hasToken}}
+                    </div>
+                    <div class="settings-button-group">
+                        {{#hasToken}}
+                            <button class="settings-button warning totp-delete">{{#t}}Disable{{/t}}</button>
+                        {{/hasToken}}
+                        {{^hasToken}}
+                            <button class="settings-button secondary totp-create">{{#t}}Enable{{/t}}</button>
+                        {{/hasToken}}
+                    </div>
+                </li>
+                {{#hasToken}}
+                    <!-- This will be enabled in https://github.com/mozilla/fxa-content-server/issues/5865 -->
+                    <!--<a class="show-recovery-codes">{{#t}}Show recovery codes{{/t}}</a>-->
+                {{/hasToken}}
+            </ul>
+
+            <div id="totp" class="totp-details hidden">
+                <div class="qr-image-container">
+                    <img class="qr-image"/>
+                </div>
+
+                <p class="setup-description">
+                    {{#t}}Scan the QR code in your app and confirm it is setup correctly by entering the security code it provides.{{/t}}
+                    <a class="show-code-link">{{#t}}Can't scan code?{{/t}}</a>
+                </p>
+
+                <div class="manual-code hidden">
+                    <p class="setup-description">{{#t}}Manually enter this secret key into your authentication app:{{/t}}</p>
```

Comment:
this seems too close to the placeholder:

![image](https://user-images.githubusercontent.com/128755/37054218-3f070bf2-214c-11e8-80cf-2311f7b6ae9a.png)